notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        column_transformations=[
</pre><p> </p><p>Gets warning: "DeprecationWarning: consider using column_specs instead. column_transformations will be deprecated in the future.</p><p> column_specs, column_transformations". Please update with column_specs</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>done</p>